cli/flox/src/utils/errors.rs

```diff
@@ -312,6 +313,16 @@ pub fn format_core_error(err: &CoreEnvironmentError) -> String {
 
             Please enable the catalog feature and try again.
         "},
+        CoreEnvironmentError::UpgradeFailedCatalog(err) => match err {
+            UpgradeError::PkgNotFound(err) => err.to_string(),
+            UpgradeError::NonEmptyNamedGroup { pkg, group } => formatdoc! {"
+                '{pkg}' is a package in the group '{group}' with multiple packages.
+                To upgrade the group, specify the group name:
+                    $ flox upgrade {group}
+                To upgrade all packages, run:
+                    $ flox upgrade
+            "},
```

Comment:
question: 3rd option is to add the package to a new group, should this be included here too?

```suggestion
            UpgradeError::NonEmptyNamedGroup { pkg, group } => formatdoc! {"
                '{pkg}' is a package in the group '{group}' with other packages
                and can not be upgraded individually.
                
                To upgrade the group, specify the group name:
                    $ flox upgrade {group}
                To upgrade all packages, run:
                    $ flox upgrade
            "},
```

Reply:
Again, this is the same error you get now from pkgdb, so I'm going to leave this how it is.